database/tables/workspace_logs.pg

```diff
@@ -0,0 +1,12 @@
+columns
+    date: timestamp with time zone default now()
+    id: bigint not null default nextval('workspace_logs_id_seq'::regclass)
+    level: smallint
```

Comment:
Let's make a `log_level` enum for this. With values of `error`, `warn`, `info`, `debug`.

Reply:
Added as `enum_workspace_log_log_level` -- name feels clunky, but I think it's the right PL convention?